ramls/counterreports.raml

```diff
@@ -1,13 +1,14 @@
 #%RAML 1.0
 title: Counter Reports
-version: v2.1
+version: v3.0
 baseUri: http://localhost/mod-erm-usage
 
 documentation:
   - title: mod-erm-usage API
     content: This documents the API calls that can be made to query and manage counter reports
 
 types:
+  counterReportDocument: !include ./schemas/counterreport_document.json
```

Comment:
Makes sense. The method now expects an entitiy of type CounterReportDocument instead of an InputStream. This also solves your mentioned issue with `requestedBytesArray`.